pkg/gadgets/trace/dns/types/dns.go

```diff
@@ -44,6 +47,21 @@ func GetColumns() *columns.Columns[Event] {
 	col, _ := cols.GetColumn("container")
 	col.Visible = false
 
+	cols.MustSetExtractor("latency", func(event *Event) string {
```

Comment:
Sounds good to me. Maybe we could mark latency as "hidden" and let the user opt-in using `-o custom-columns`?

Reply:
I agree to hide it by default in the meanwhile. We should not add a lot of columns by default, otherwise it'll need a very large terminal to see it correctly.